app/controllers/api/v1/serialisers.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
```

Comment:
It might've been useful if the refactor of the V1 serialisation was in a separate commit, just because the commit message only references V2. I can see that the V1 specs didn't change however 👍

Reply:
yeah, that's fair enough - I got to the point of adding the v2 APIs and then noticed the serialisation logic was on the model so broke it out at that point